modules/core/src/main/java/org/apache/ignite/hlc/HybridTimestamp.java

```diff
@@ -17,13 +17,14 @@
 
 package org.apache.ignite.hlc;
 
+import java.io.Serializable;
 import org.apache.ignite.internal.tostring.S;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * A hybrid timestamp that combines physical clock and logical clock.
  */
-public class HybridTimestamp implements Comparable<HybridTimestamp> {
+public class HybridTimestamp implements Comparable<HybridTimestamp>, Serializable {
```

Comment:
Why? @Marshallable is enough for proper message fields serialization.

Reply:
It's necessary for `ByteUtils#toBytes`.